influence_benchmark/RL/run_KTO_iteration.py

```diff
@@ -17,6 +17,21 @@
 ), f"If you have an accelerate config file, it will overwrite our defaults {cache_dir}"
 
 
+def make_system_prompt_user_message(messages_in):
+    """
+    Make the system prompt user message for gemma.
+    """
+    if messages_in[0]["role"] == "system":
+        messages_in[0]["role"] = "user"
+        new_content = (
+            f"<Instructions>\n\n {messages_in[0]['content']}</Instructions>\n\n{messages_in[1]['content']}\n\n"
+        )
+        messages_in[1]["content"] = new_content
+        del messages_in[0]
+
+    return messages_in
```

Comment:
Have this be a static method of the HFBackend class?

Reply:
Then import HFBackend with a local import with the other imports from influence_benchmark in this file

Reply:
Fixed in #144